doc/us-en/setup/Install-OpenEdge-on-Darwin.md

```diff
@@ -1,2 +1,102 @@
+# Install OpenEdge On Darwin
 
-> TO DO(@sherlockblaze)
\ No newline at end of file
+> OpenEdge is mainly developed in Go programming language and supports two startup modes: ***docker*** container mode and ***native*** process mode.
+
+This document focuses on the installation and configuration of the environment required for OpenEdge and the rapid deployment of OpenEdge on the Darwin system.
+
+## Environment Configuration
+
+### Install Docker
+
+> OpenEdge offers two startup modes. To start using ***docker*** container mode (recommended), you need to complete the docker installation first.
+
+**Notice:**
+
++ The official Dockerfile is offered for multi-stage builds. If you need to build the relevant image yourself, The version of docker you installed should be above 17.05.
++ The production environment can run the image using a lower version of Docker, which is currently tested to a minimum usable version of 12.0.
++ According to the [Official Release Log](https://docs.docker.com/engine/release-notes/#18092), the version of docker version < 18.09.2 has some security implications. It is recommended to install/update the Docker version to 18.09. 2 and above.
+
+Go to [official page](https://hub.docker.com/editions/community/docker-ce-desktop-mac) to download the .dmg file you need. Once done, double-click to open and drag Docker into the Application folder.
+
+![Install On Darwin](../../images/setup/docker-install-on-mac.png)
+
+View the version of installed Docker:
+
+```shell
+docker version
+```
+
+**For more details, please see the [official documentation](https://docs.docker.com/install/).**
+
+### Install Python2.7 and Python runtime dependency package
+
+> + OpenEdge provides Python Runtime, which supports running code written in Python version 2.7. If you plan to start using ***native*** process mode, you need to install Python 2.7 and the package you depend on. If you plan to start in ***docker*** container mode, you do not need to perform the following steps.
+
+Install by using HomeBrew(Recommended):
+
+```shell
+/usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"  // Install HomeBrew
+brew install python@2
+pip install protobuf grpcio
+```
+
+***Notice***: Execute the following command to check the installed Python version:
```

Comment:
***Notice***: Execute the following command to check the installed version of Python: